deployment/README.md

```diff
@@ -43,4 +43,4 @@ docker-compose down
 ## Troubleshooting
 
 - Make sure you are starting all the Docker containers in a clean environment. This means before starting the services make sure all containers are removed, all images are removed, and all volumes are removed.
-- If you are having trouble and cannot find the answer here, please contact someone from the Aerie team via the [#mpsa-aerie-users](https://app.slack.com/client/T024LMMEZ/C0163E42UBF) Slack channel.
```

Comment:
It might just have been a long day, but I can't work out what changed on this line? Ultimately, I would prefer the reference to the slack channel to be removed. I hadn't gotten around to this so I appreciate you contributing.

Reply:
Not to be pedantic, can you please squash these two commits?

Reply:
I was scratching my head at this as well. He just removed a newline. I'm also for removing this line in a future doc update.